services/mentor-api/tests/features/mentors_answer_questions.feature

```diff
@@ -3,9 +3,10 @@ Feature: Mentors answer questions
   Scenario Outline: query to a mentor receives ideal response
     Given a request url http://localhost:5000/mentor-api/questions
         And request parameters
-          | param     | value         |
-          | query     | <query>       |
-          | mentor    | <mentor>      |
+          | param                          | value         |
+          | query                          | <query>       |
+          | mentor                         | <mentor>      |
+          | canned_question_match_disabled | <canned_question_match_disabled> |
```

Comment:
I think let's not include distinctions about `disable-canned` in this test. We should make a new distinct test that tests canned-question-disabled-true (the default); everywhere else, let's set `canned_question_match_disabled` false so that we're actually testing the classifier. So you can then also take it out of the examples (just hard code it false)